systemtests/bin/setup_download_tests.py

```diff
@@ -117,9 +117,7 @@ def write_list_of_sym_filenames_to_csv(sym_file_type_to_filename, outputdir):
     date = datetime.datetime.now().strftime("%Y-%m-%d")
     csv_rows.append([f"# File built: {date}"])
     csv_rows.append(["# Format: symfile", "expected status code", "bucket"])
-    csv_rows.append(["@ECHO NOTE: None of these should take over 1000ms to download."])
```

Comment:
Because this file is generated automatically, it can contain `xul.sym` or other _very large_ symbols files which take longer than 1000ms (1s) to download. I couldn't think of a better note, so I removed this note and the `@ECHO` handling.

Reply:
I don't think we need a _very large_ symbol file for the test for bug #1655944. I think the chunked-transfer encoding will trigger for most of the symbols files we can build. The way jwhitlock reproduced the issue was with a zip file that was 120,684 bytes. That's not that big. The zip files I have in my `systemtests/data/zips/` dir are all > 26mb. We can adjust the code to look for a zip file that's > 120kb and use the first one and if there aren't any, have it raise an error.

Filtering for smaller files is a good idea. I can fix that in this PR.